R/convertFlowCore.R

```diff
@@ -0,0 +1,103 @@
+library("stats")
+
+#' Convert a gate to flowCore
+#'
+#' Converts a CellEngine gate ("PolygonGate", "EllipseGate", or "RectangleGate") to
+#' its flowCore analogue.
+#'
+#' @param gate The CellEngine gate to be converted
+convertToFlowCore <- function (gate) {
```

Comment:
I think this should be added to NAMESPACE also.

Reply:
It's very weird that it's not; Roxygen should put it there :thinking: